### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by remcohaszing**
All caret ranges are now defined as `^x.0.0`.

A small ESLint issue was fixed, which was reported by the updates ESLint configuration.

### CODE REVIEW

**Review by yaabed — APPROVED** [review]

**Review by yazaabed — APPROVED** [review]

**Comment by yazaabed** [comment]
Fancy Webpack 5 😮

File: src/languageFeatures.ts [hunk 1]
```diff
@@ -113,7 +113,9 @@ export function createDiagnosticsAdapter(
     }
   });
 
-  editor.getModels().forEach(onModelAdd);
+  for (const model of editor.getModels()) {
```
Comment (fleon) [inline src/languageFeatures.ts:116]: I understand this is better, but was it causing any bug earlier that this had to be changed?

**Review by fleon — APPROVED** [review]

**Comment by remcohaszing** [comment]
> Fancy Webpack 5 :open_mouth:

It was already webpack 5 :wink: This webpack update does allow us to upgrade `webpack-dev-server` though.

File: src/languageFeatures.ts (hunk 1 above)
Comment (remcohaszing) [reply src/languageFeatures.ts:116]: I enabled `unicorn/no-for-each` in my ESLint configuration. It’s (almost) purely stylistic.

**Pull request merged**